frontend/src/testing/pushapi-mock.js

```diff
@@ -1,28 +1,21 @@
 /**
- * @author Stefan Dimitrov (stefan.dimitrov@clouway.com).
- */
+  * @author Stefan Dimitrov (stefan.dimitrov@clouway.com).
+  */
+ angular.module('clouway-push', [])
 
-angular.module('clouway-push')
+   .config(function ($provide, pushApiProvider, $httpBackend) {
```

Comment:
When other users write tests in their app of logic that uses the pushApi, they will not need to expect or stub $httpBackend requests, and hardcode any urls, but just make the stubbed pushApi flush its backend requests.

It's still something extra to remember to do, but its one simple thing as opposed to 2 or 3 a bit more complicated things.